lib/MongoDB/Collection.pm

```diff
@@ -727,6 +727,48 @@ sub find_one {
     )->execute->next;
 }
 
+=method find_id
```

Comment:
I think shorter documentation that references `find_one` would be fine.

Minor note: for a POD `C<...>` expression, if there is a `>` inside, you need to write it in a longer form like this (delimited with doubled brackets and spaces): `C<< ... >>`.

For example: `C<< _id => $id >>`